recipes/glad/2.x/conanfile.py

```diff
@@ -0,0 +1,138 @@
+import os
+import sys
+from pathlib import PurePosixPath
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, collect_libs, copy, export_conandata_patches, get, rmdir
+from conan.errors import ConanInvalidConfiguration
+
+class GladConan(ConanFile):
+    name = "glad"
+    description = "Multi-Language GL/GLES/EGL/GLX/WGL Loader-Generator based on the official specs."
+    topics = ("opengl",)
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Dav1dde/glad"
+    license = "MIT"
+    settings = "os", "compiler", "build_type", "arch"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "no_loader": [True, False],
+        "spec": ["gl", "egl", "glx", "wgl"], # Not relevant for 2.x. A spec will be included unless its version is None
```

Comment:
I've left this in because it's in the `0.x` recipe - but specs aren't mutually exclusive - in fact a GLES spec won't work without also including egl.

Would be good if we can just remove this option?